### PR TITLE
Clean up Fabric Version Methodology

### PR DESCRIPTION
**Pull request by btl5037**
Signed-off-by: Brett Logan <brett.t.logan@ibm.com>

### CODE REVIEW

File: fabric-chaincode-integration-test/getDockerImages.sh [hunk 1]
```diff
@@ -1,43 +1,22 @@
...
+version=${FABRIC_VERSION:-2.1}
+artifactory_url=hyperledger-fabric.jfrog.io
 
-###############################################################
-# Pull and Tag the fabric and fabric-ca images from Artifactory
-###############################################################
-echo "Fetching images from Artifactory"
-ARTIFACTORY_URL=hyperledger-fabric.jfrog.io
-ORG_NAME="hyperledger"
+for image in peer orderer ca tools orderer ccenv; do
```
Comment (btl5037) [inline fabric-chaincode-integration-test/getDockerImages.sh:12]: Do you actually require `ccenv` for anything? Or can it be removed

File: fabric-chaincode-integration-test/src/test/resources/first-network/base/peer-base.yaml [hunk 2]
```diff
@@ -22,7 +22,8 @@ services:
       - CORE_PEER_TLS_CERT_FILE=/etc/hyperledger/fabric/tls/server.crt
       - CORE_PEER_TLS_KEY_FILE=/etc/hyperledger/fabric/tls/server.key
       - CORE_PEER_TLS_ROOTCERT_FILE=/etc/hyperledger/fabric/tls/ca.crt
-      - CORE_CHAINCODE_JAVA_RUNTIME=hyperledger/fabric-javaenv:amd64-latest
       - CORE_CHAINCODE_EXECUTETIMEOUT=400s
+      - CORE_CHAINCODE_BUILDER=hyperledger/fabric-ccenv
```
Comment (btl5037) [inline fabric-chaincode-integration-test/src/test/resources/first-network/base/peer-base.yaml:26]: I will remove this if we agree to remove `ccenv` from the pull script

File: fabric-chaincode-integration-test/getDockerImages.sh (hunk 1 above)
Comment (mbwhite) [reply fabric-chaincode-integration-test/getDockerImages.sh:12]: I think it's a hangover from the ancestor of this script so it can be removed.

**Comment by btl5037** [comment]
> ccenv is redudant here... so can be removed....
> Happy with the rest

Removed ccenv, thanks Matt

**Pull request merged**